base/dcp_sharded.go

```diff
@@ -536,6 +592,12 @@ func (l *importHeartbeatListener) subscribeNodeChanges() error {
 					}
 				}
 
+				if l.ctx.LithiumCompat.IsTrue() && minCompatVersion >= compatVersionHelium {
```

Comment:
Similar question to the above - I don't think we're even registering the importHeartbeatListener if LithiumCompat.IsTrue(), so does this code get used outside of the unit test?